### PR TITLE
Add featured project

### LINKED ISSUE #9: Show featured project
As suggested by this guy, and other people, 
Screenshot of message saying "Is there a way to embed the thumbnail of featured project here?

### PR DESCRIPTION
**Pull request by moeenio**
Adding a basic featured project display, a long requested feature.

Erased Andreto's branch by accident (sorry mate).

Fixes #9

### CODE REVIEW

**Review by Andreto — APPROVED** [review]
Looks good!
Sorry it took so long, I'm on vacation and not s active.

**Comment by moeenio** [comment]
That's fine !

**Comment by moeenio** [comment]
That's not beautiful, but, well, let's stick with that.
@MasterOfTheTiger, can you leave a review ?

File: js/main.js [hunk 1]
```diff
@@ -78,4 +95,18 @@ req.onreadystatechange = function() {
...
+//reqFP.send();
+//reqFP.onreadystatechange = function() {
+//  if (reqFP.readyState === 4 && reqFP.status === 200) {
+//    var parsedrespFP = JSON.parse(reqFP.responseText);
+//    console.log(parsedrespFP.featured_project);
+//    featuredImg.src = "https://cdn2.scratch.mit.edu/get_image/project/" + parsedrespFP.featured_project + "_282x210.png";
+//    featuredA.href = "https://scratch.mit.edu/projects/" + parsedrespFP.featured_project;
+//    featuredTitle.innerHTML = parsedrespFP.featured_project_data.title;
+//  }
+//}
```
Comment (MasterOfTheTiger) [inline js/main.js:112]: Everything looks good, but what is the last commented section for?

**Review by MasterOfTheTiger — COMMENTED** [review]
All looks good except I am wondering about the last part (see related comment)

File: js/main.js (hunk 1 above)
Comment (moeenio) [reply js/main.js:112]: It's the original code of the second request, which is now wrapped in the first request's function, but I kept below for... some reason.

Comment (MasterOfTheTiger) [reply js/main.js:112]: There doesn't seem to be any reason to keeping it, but if you have a reason, keep it. Otherwise, I approve the merge.

Comment (moeenio) [reply js/main.js:112]: Well, please click Approve.

**Review by MasterOfTheTiger — APPROVED** [review]
Not sure how much this is worth (or why you had me approve it), but here's my approval.

**Comment by moeenio** [comment]
Gonna reach out for testers.

**Comment by moeenio** [comment]
The wait is long enough, I think we need to roll this out ! Merging now!

**Comment by githubfan** [comment]
Yay! Well done Locness and the team
:)

**Comment by moeenio** [comment]
Thanks @githubfan !

**Comment by githubfan** [comment]
> Thanks @githubfan !

Soo... what should I do now with my original code?

**Comment by moeenio** [comment]
Just merged the pull request, we just need to wait a few minutes and it will be live for everyone!

**Comment by githubfan** [comment]
> Just merged the pull request, we just need to wait a few minutes and it will be live for everyone!

That's great!

**Comment by githubfan** [comment]
> > Just merged the pull request, we just need to wait a few minutes and it will be live for everyone!
> 
> That's great!

I'll try to promote it on my website!

**Comment by moeenio** [comment]
It's showing up for me, but there is a bug with griffpatch

**Comment by moeenio** [comment]
I will make a new studio so everyone gets informed :) Also I will do a changelog with https://headwayapp.co

**Comment by githubfan** [comment]
> I will make a new studio so everyone gets informed :) Also I will do a changelog with https://headwayapp.co

Great! When will that roll out?

**Comment by moeenio** [comment]
Working on it :)

**Comment by githubfan** [comment]
> Working on it :)
Nice! I'll wait for an invite to the new studio. Please also invite my scratch alt at @joshuay2_alt

**Comment by githubfan** [comment]
@locness3 Have you finished yet?

**Comment by githubfan** [comment]
*That's Great!*

*Coding Master*

*WEB DEVELOPER / BLOGGER*

<https://www.facebook.com/>
<https://twitter.com/>
<https://instagram.com/> .


Drop me a line_ <https://githubfan.github.io>

*Coding Master_Web Developer / Writer*

**Comment by moeenio** [comment]
No. I'm doing other things in parallel.

**Comment by moeenio** [comment]
Sorry @githubfan, it's taking long because I'm making GIF recordings to put in it.

**Comment by githubfan** [comment]
> Sorry @githubfan, it's taking long because I'm making GIF recordings to put in it.

Have we done yet?

**Comment by moeenio** [comment]
Yes!

**Comment by githubfan** [comment]
> Yes!

Yay!

**Comment by locness3** [comment]
Haven't you seen it already ?

**Comment by moeenio** [comment]
@Andreto Oh sorry, I didn't put you as commit co-author :/ You deserve all credit though

**Pull request merged**